pkg/graph/schema/types/mto_solution.graphql

```diff
@@ -46,12 +44,18 @@ input MTOSolutionChanges @goModel(model: "map[string]interface{}") {
   pocEmail: String
 }
 
+input MTOMilestoneLinks {
+  milestoneIDs: [UUID!]
+}
+
 extend type Mutation {
   createMTOSolutionCustom(modelPlanID: UUID!, solutionType: MTOSolutionType!,  name: String!, pocName: String!, pocEmail: String!): MTOSolution!
   @hasRole(role: MINT_USER)
   createMTOSolutionCommon(modelPlanID: UUID!, key: MTOCommonSolutionKey!, milestonesToLink: [UUID!]): MTOSolution!
   @hasRole(role: MINT_USER)
-  updateMTOSolution(id: UUID!, changes: MTOSolutionChanges!): MTOSolution!
+  updateMTOSolution(id: UUID!, changes: MTOSolutionChanges!, milestoneLinks: MTOMilestoneLinks): MTOSolution!
+  @hasRole(role: MINT_USER)
+  mtoSolutionUpdateLinkedMilestones(id: UUID!, milestoneLinks: MTOMilestoneLinks!): [MTOMilestone!]
```

Comment:
I don't think this endpoint is needed, just the update endpoint. I think it should be removed unless needed by the frontend. It can be added later if needed

Reply:
This is the same behavior as we added for linking solutions to milestones. In the interest of consistency I think it's best to keep the same structure for parity. Additionally, it allows for more atomic endpoint testing. Here you can validate specific behavior without concern for integration into the update mutation.